drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -2265,4 +2343,12 @@ void RasterizerCanvasGLES3::finalize() {
 }
 
 RasterizerCanvasGLES3::RasterizerCanvasGLES3() {
+	// Not needed (a priori) on GLES devices
+	use_nvidia_rect_workaround = false;
+
+#ifdef GLES_OVER_GL
+	if (!Engine::get_singleton()->is_editor_hint()) {
```

Comment:
Why exclude it from the editor? Users might still see flickering in their TileMaps when editing them.

Reply:
I originally didn't, but the IDE seemed a little slower typing scripts with it set on in the editor. Maybe I'm imagining it. I'm ok with leaving it on in the editor too, I'll change this.